.github/workflows/tests.yml

```diff
@@ -128,9 +127,7 @@ jobs:
           else
             pip install --upgrade Cython numpy==${{ matrix.numpy }} pytest pytest-cov coveralls;
           fi
-          if [ -n "${{ matrix.numpy }}" ]; then
```

Comment:
Whoops, I suppose the intention was that this check was `matrix.matplotlib` instead of `matrix.numpy` ..

Reply:
I see! Will update accordingly a little later.